tests/test_downloader.py

```diff
@@ -0,0 +1,26 @@
+import unittest

```

Comment:
`tests/test_downloader.py` structure does not tell me many info, try to think about hierarchy
```bash
tests/
- units/
-- test_history_downloader.py...
```

Reply:
Yes, it's changed now, should be better